testing-standard.md

```diff
@@ -40,7 +41,7 @@ expect(myStub).to.have.been.called();
 
 
 ## 1.2 Good Test Folders
-When addind new tests to projects, test files should sit in a folder hierarchy that mimics the hierarchy fo the source code. The test files themselves should be named similarly to the source files, but with `test` at the beginning of the file name. For example, if we have a source file `src/views/checkoutView.js` we should have a test file `test/unit/views/checkoutView.js` or similar. This makes finding tests an easy and logical process.
+When adding new tests to projects, test files should sit in a folder hierarchy that mimics the hierarchy fo the source code. The test files themselves should be named similarly to the source files, but with `test` at the beginning of the file name. For example, if we have a source file `src/views/checkoutView.js` we should have a test file `test/unit/views/checkoutView.js` or similar. This makes finding tests an easy and logical process.
```

Comment:
i'm totally against the `_test` suffix, it's gross. 
I also think prefixing with `test` is redundant because it's already in a `test` folder but i'm not too fussed either way.